progress.py

```diff
@@ -95,7 +95,7 @@ def GetNonMatchingSize(path):
 ovl -= nonMatchingASMOvl
 
 bootSize = 31408 # decompilable code only
-codeSize = 1000000 # decompilable code only (1.00mb)
+codeSize = 999984 # decompilable code only (1.00mb)
```

Comment:
I think the 1.00mb comment should go
```suggestion
codeSize = 999984 # decompilable code only
```